config/bootstrap.php

```diff
@@ -33,11 +33,21 @@
  */
 require CORE_PATH . 'config' . DS . 'bootstrap.php';
 
+// You can remove this if you are confident that your PHP version is sufficient.
+if (version_compare(PHP_VERSION, '5.5.9') < 0) {
```

Comment:
@dereuromark https://github.com/cakephp/app/blob/master/composer.json#L8 might not be required btw because it is already checked by cakephp/cakephp/composer.json
Remove?